recipe/meta.yaml

```diff
@@ -17,18 +17,21 @@ requirements:
     - setuptools
     - cython >=0.23
     - python
-    - six >=1.4
+    - six >=1.7.3
     - numpy x.x
-    - scipy >=0.9
+    - scipy >=0.17
+    - numpydoc >=0.6
   run:
     - python
-    - six >=1.4
-    - numpy x.x
```

Comment:
Oh please, your response time is amazing. =) I just played with the re-render so that it gave me back that nice build matrix. =)

Re further questions, from a usability perspective, `- numpy x.x` is super cryptic, *and*, even allowing for it, imho it should only be required in one place, which is more fittingly the `build` section.